keymaps/git.cson

```diff
@@ -3,6 +3,19 @@
   'ctrl-shift-8': 'github:toggle-github-tab'
   'ctrl-9': 'github:toggle-git-tab-focus'
   'ctrl-shift-9': 'github:toggle-git-tab'
+  'alt-g enter': 'github:commit'
+  'alt-g f': 'github:fetch'
+  'alt-g F': 'github:pull'
```

Comment:
@lee-dohm Have we changed the normalization scheme in the `atom-keymap` source code? If not, there's still places where we'll display upcased shift-modified keycaps in the editor, as in `alt-shift-S` instead of `alt-shift-s`.